Pipfile.lock

```diff
@@ -1654,11 +1651,11 @@
         },
         "dill": {
             "hashes": [
-                "sha256:a07ffd2351b8c678dfc4a856a3005f8067aea51d6ba6c700796a4d9e280f39f0",
-                "sha256:e5db55f3687856d8fbdab002ed78544e1c4559a130302693d839dfe8f93f2373"
+                "sha256:76b122c08ef4ce2eedcd4d1abd8e641114bfc6c2867f49f3c41facf65bf19f5e",
+                "sha256:cc1c8b182eb3013e24bd475ff2e9295af86c1a38eb1aff128dac8962a9ce3c03"
             ],
-            "markers": "python_version < '3.11'",
-            "version": "==0.3.6"
+            "index": "pypi",
+            "version": "==0.3.7"
```

Comment:
I get:
```
            "markers": "python_version >= '3.11'",
            "version": "==0.3.7"
```
if I run `pipenv install -d` on python 3.11.4

Reply:
I'm on the same version and I ran the same cmd, I'll never fully understand python 😅

Reply:
Did you create a clean pipenv? That's what I did.

Reply:
Okay I assumed the python version in `requires` worked like in node where you can just specify the typescript version and it will handle everything. Seems I was wrong, remade a new venv with 3.11, deleted the lock file and ran `pipenv update` again, now everything should be the same as yours!